src/fw/board/boards/board_asterix.c

```diff
@@ -152,10 +152,13 @@ void board_early_init(void) {
   nrf_gpio_pin_set(15);
   nrf_gpio_pin_set(16);
 
-  /* TODO: check that LFCLK actually comes up */
+  nrf_clock_lf_src_set(NRF_CLOCK, NRF_CLOCK_LFCLK_XTAL);
   nrf_clock_event_clear(NRF_CLOCK, NRF_CLOCK_EVENT_LFCLKSTARTED);
-  nrf_clock_int_enable(NRF_CLOCK, NRF_CLOCK_INT_LF_STARTED_MASK);
   nrf_clock_task_trigger(NRF_CLOCK, NRF_CLOCK_TASK_LFCLKSTART);
+  while (!nrf_clock_event_check(NRF_CLOCK, NRF_CLOCK_EVENT_LFCLKSTARTED)) {// &&
+         /*!nrf_clock_lf_is_running(NRF_CLOCK)) {*/
```

Comment:
Is the comment here a leftover or supposed to be explaining something?

Reply:
ooops